src/library/LocalLoopSaturationDirectActingMax.py

```diff
@@ -0,0 +1,64 @@
+"""
+## Local Loop Performance Verification - Direct Acting Loop Actuator Maximum Saturation
+
+### Description
+
+This verification checks that a direct acting control loop would saturate its actuator to maximum when the error is consistently above the set point.
+
+### Verification logic
+
+If the sensed data values are consistently above its set point, and after a default of 1 hour, the control command is still not saturated to maximum, then the verification fails; Otherwise, it passes.
+
+### Data requirements
+
+- feedback_sensor: feedback sensor reading of the subject to be controlled towards a set point
+- set_point: set point value
+- cmd : control command
+- cmd_max: control command range maximum value
+
+"""
+
+import pandas as pd
+from checklib import RuleCheckBase
+
+
+class LocalLoopSaturationDirectActingMax(RuleCheckBase):
+    points = ["feedback_sensor", "set_point", "cmd", "cmd_max"]
+
+    def saturation_flag(self, t):
+        if 0 <= t["cmd_max"] - t["cmd"] <= 0.01:
+            return True
+        else:
+            return False
+
+    def err_flag(self, t):
+        if t["feedback_sensor"] > t["set_point"]:
```

Comment:
Same comment as above for the command. I doubt it but still wanted to raise the question in case you think that it might be an issue.

Reply:
To me this is a different issue from the comment earlier. Because this is a direct acting controller, then saturating towards maximum is the expected behavior when the sensor reading is constantly above the set point, not the other way around.